src/components/ContextBar.test.js

```diff
@@ -0,0 +1,610 @@
+import '@testing-library/jest-dom'
```

Comment:
the location of ContextBar.js has moved to a more dedicated area owned by IA team:

src/pages/workspaces/workspace/analysis/ContextBar.js

Also, because the UIWG has approved the convention of co-locating test files to be right next to the file being tested, this .test.js file should move over to that new location as well.

Reply:
@jdcanas has moved this file in his PR: https://github.com/DataBiosphere/terra-ui/pull/3422